### ISSUE TITLE
Unusable settings icon in In-App Browser

### ISSUE THREAD

**Issue by arshubham** [artemanufrij/webpin#11]
![screenshot from 2017-08-10 20 15 41](https://user-images.githubusercontent.com/14235191/29176342-334e78c2-7e09-11e7-88b3-87134eb9a883.jpeg)

**Comment by artemanufrij** [comment]
I'm going to change behavior for external links #10 . It should be open in default browser instead webpin

**Comment by artemanufrij** [comment]
hey @arshubham, it would be nice if you could test current master branch. I changed "external link" behavior.

**Comment by arshubham** [comment]
Thanks for the quick fix. It works as expected. Tested with multiple websites. The current method for opening external links is fine for the external websites, but I think the application should prompt the user when it is about to open apps other than the browser. 
For instance I accidentally clicked a mailto: link and it opened Thunderbird after 10 seconds. I kept wondering when did I open the application.
